File: Dataset/Leetcode/train/9/43.java
class Solution {
    public boolean XXX(int x) {
        int a = x;
        if (x < 0) {
            return false;
        }
        if (x < 10) {
            return true;
        }
        if (x % 10 == 0) {
            return false; 
        }
        int t = 0;
        while (x > 0) {
            t = t * 10 + x % 10;
            x /= 10;
        }
       return t == a;
    }
}

File: Dataset/Leetcode/train/75/43.py
class Solution:
    def XXX(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        red = 0
        white = 0
        for i in range(len(nums)):
            if nums[i] == 0:
                red += 1
            elif nums[i] == 1:
                white += 1
        for i in range(len(nums)):
            if i < red:
                nums[i] = 0
            elif i < (red+white):
                nums[i] = 1
            else:
                nums[i] = 2

